src/Libraries/OrchardCore.Commerce.AddressDataType/Regions.cs

```diff
@@ -10,27 +10,25 @@ public static class Regions
     /// <summary>
     /// Gets the list of regions.
     /// </summary>
-    public static IList<RegionInfo> All { get; } =
+    [Obsolete("Use service! This is a temporary attribute.")]
+    public static IList<Region> All { get; } =
         CultureInfo
             .GetCultures(CultureTypes.SpecificCultures)
+            .Where(culture => !culture.Equals(CultureInfo.InvariantCulture) && !culture.IsNeutralCulture)
```

Comment:
Hmm. I think this is not needed, since we are using `.GetCultures(CultureTypes.SpecificCultures)` instead of `.GetCultures(CultureTypes.AllCultures)`. I checked the results of `.GetCultures(CultureTypes.SpecificCultures)` and I couldn't find any cultures with `InvariantCulture`, or `IsNeutralCulture`.

Also with and without this `Where` condition, `Regions.All` have 135 members anyway.